feat: fcc-todo-app (#51367)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-localstorage-by-building-a-todo/64ec89ee549ecf802de2b3e2.md

---
id: 64ec89ee549ecf802de2b3e2
title: Step 14
challengeType: 0
dashedName: step-14
---

# --description--

> Get the values of the `titleInput`, `dateInput`, and `descriptionInput` input fields. Save them in the properties `title`, `date`, and `description`.

Reviewer: ```suggestion
Retrieve the values from the `titleInput`, `dateInput`, and `descriptionInput` fields, and then save them in the properties `title`, `date`, and `description` of the `taskObj` object.
```
